auth-tokens-filter/src/main/java/com/palantir/tokens/auth/http/BearerTokenLoggingFilter.java

```diff
@@ -41,17 +43,34 @@
     public static final String SESSION_ID_KEY = "sessionId";
     public static final String TOKEN_ID_KEY = "tokenId";
 
+    private final String cookieAuthKey;
+
+    public BearerTokenLoggingFilter(String cookieAuthKey) {
+        this.cookieAuthKey = cookieAuthKey;
```

Comment:
this is a break, should be optional/default with a no-arg ctor.

Reply:
ok